src/geoschem_run_scripts/run_jacobian_simulations.sh

```diff
@@ -30,12 +30,33 @@ if test -f "$FILE"; then
     exit 1
 fi
 
-### Run GEOS-Chem in the directory corresponding to the cluster Id
-cd  ${RUNDIR}/{RunName}_${xstr}
-./{RunName}_${xstr}.run
-
-# save the exit code of the jacobian simulation cmd
-retVal=$?
+if {ReDoJacobian}; then
+    # check for last conc file
+    # it has only 1 timestep, so we
+    # only need check its existence
+    # rerun if it is not there
+    LastConcFile=$(date -d {EndDate} +GEOSChem.SpeciesConc.%Y%m%d_0000z.nc4)
```

Comment:
Will this work for KalmanMode where there are multiple sets of jacobian simulations (one set for each iteration)? Or will it only check the final EndDate?

Reply:
Updated to work with Kalman mode. It now gets the end date from the GEOS-Chem config file, which is updated each Kalman period